tests/fast-integration/test/getting-started-guides.js

```diff
@@ -0,0 +1,359 @@
+      orderCode: "762727210",
+      consignmentCode: "76272725",
+      consignmentStatus: "PICKUP_COMPLETE",
+    };
+    await createOrder(serviceDomain, order);
+
+    await getOrders(serviceDomain, (resp) => {
+      expect(resp).to.be.an("Array").of.length(1);
+      expect(resp[0]).to.deep.eq(order);
+    });
+
+    await deletePodsByLabel(orderService, `app=${orderService}`);
+
+    await getOrders(serviceDomain, (resp) =>
+      expect(resp).to.be.an("Array").of.length(0)
+    );
+
+    // https://kyma-project.io/docs/root/getting-started/#getting-started-bind-the-redis-service-instance-to-the-microservice
+    console.log("Creating ServiceBindingUsage...");
+    await k8sDynamicApi.create(sbuObj).catch(expectNoK8sErr);
+
+    await waitForSbuReady(orderService, orderService);
+
+    // TODO @aerfio I think that this step with checking if secret exists is kinda redundant, would get rid of it, we already check if
+    const secret = await retryPromise(
+      async () => {
+        console.log("Waiting for SBU's secret to appear");
+        const sec = await k8sCoreV1Api.readNamespacedSecret(
+          orderService,
+          orderService
+        );
+        return sec;
+      },
+      10,
+      5000
+    ).catch(expectNoK8sErr);
+
+    expect(secret.body.data).to.have.property("HOST");
+    expect(secret.body.data).to.have.property("PORT");
+    expect(secret.body.data).to.have.property("REDIS_PASSWORD");
+
+    await deletePodsByLabel(orderService, `app=${orderService}`);
+    // TODO jesus christ another nasty datarace, old pods that are already being deleted are still receiving traffic
+    // we might have to introduce some function that not only deletes the pods, but also waits till they're completly gone
+    await sleep(20000);
+
+    await getOrders(serviceDomain, (resp) =>
+      expect(resp).to.be.an("Array").of.length(0)
+    );
+
+    await createOrder(serviceDomain, order);
+
+    await getOrders(serviceDomain, (resp) => {
+      expect(resp).to.be.an("Array").of.length(1);
+      expect(resp[0]).to.deep.eq(order);
+    });
+
+    await deletePodsByLabel(orderService, `app=${orderService}`);
+
+    await getOrders(
+      serviceDomain,
+      (resp) => {
+        expect(resp).to.be.an("Array").of.length(1);
+        expect(resp[0]).to.deep.eq(order);
+      },
+      20, // longer, because the pod has just been killed and it needs to start again
+      5000
+    );
+
+    // https://kyma-project.io/docs/root/getting-started/#getting-started-connect-an-external-application
+    console.log(
+      "Creating addonconfiguration, serviceinstance, application and tokenrequest"
+    );
+    await Promise.all(xfMockObjs.map((obj) => k8sDynamicApi.create(obj))).catch(
+      expectNoK8sErr
+    );
+  });
+  // https://kyma-project.io/docs/root/getting-started/#getting-started-connect-an-external-application-connect-events
+  // TODO: this step requires us to use UI, needs to be handled somehow here in nodejs
```

Comment:
You did it already in other test - you can reuse connected commerce mock here. We need to install and connect commerce mock once before other tests.